<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>httyd_question_3</class>
 <widget class="QDialog" name="httyd_question_3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Berk's Dragon Rider Test</string>
  </property>
  <widget class="QLabel" name="lbl_background_httyd">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>37</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="option_a">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>530</y>
     <width>371</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_question_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>395</y>
     <width>911</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="option_b">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>371</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_continue">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>80</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="httyd_progress_bar">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>1021</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Symbol</family>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::Direction::TopToBottom</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lives_label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>630</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Lives :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lives_display">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>640</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
